### ISSUE TITLE
Home-Assistant won't load automations

### ISSUE THREAD

**Issue by jardous** [getumbrel/umbrel-apps#29]
After today's upgrade to 0.5.0 my home-automation app does not seem to fully work

Automations won't load.

Newly created automations are not added to the empty list.

It seems there is an access right issue:

`umbrel@umbrel:~/homeassistant $ ls -la 
total 387752
drwxr-xr-x 10 umbrel umbrel      4096 Jun 11 21:07 .
drwxr-xr-x  8 umbrel umbrel      4096 Jun 11 20:50 ..
-rw-r--r--  1 root   root        4615 Jun 11 21:07 automations.yaml
-rw-r--r--  1 umbrel umbrel      4333 Jun 11 20:57 automations.yaml.orig
drwxr-xr-x  2 root   root        4096 Jun 11 20:51 backups
drwxr-xr-x  4 umbrel umbrel      4096 Aug 22  2021 blueprints
drwxr-xr-x  2 umbrel umbrel      4096 Dec  3  2021 .cloud
-rw-r--r--  1 umbrel umbrel       968 Jan 17 20:24 configuration.yaml
drwxr-xr-x  3 umbrel umbrel      4096 Sep  5  2021 custom_components
drwxr-xr-x  2 umbrel umbrel      4096 Aug 22  2021 deps
-rw-r--r--  1 umbrel umbrel         0 Aug 22  2021 groups.yaml
-rw-r--r--  1 umbrel umbrel         8 Jun 11 20:32 .HA_VERSION
-rw-r--r--  1 root   root       14548 Jun 11 21:02 home-assistant.log
-rw-r--r--  1 umbrel umbrel     18232 Jun 11 21:02 home-assistant.log.1
-rw-r--r--  1 root   root           0 Jun 11 21:02 home-assistant.log.fault
-rw-r--r--  1 umbrel umbrel 392753152 Jun 11 21:02 home-assistant_v2.db
-rw-r--r--  1 umbrel umbrel     32768 Jun 11 21:07 home-assistant_v2.db-shm
-rw-r--r--  1 umbrel umbrel   3675072 Jun 11 21:07 home-assistant_v2.db-wal
-rw-r--r--  1 umbrel umbrel        68 Aug 22  2021 known_devices.yaml
-rw-r--r--  1 umbrel umbrel         3 Aug 22  2021 scenes.yaml
-rw-r--r--  1 umbrel umbrel         0 Aug 22  2021 scripts.yaml
-rw-r--r--  1 umbrel umbrel       161 Aug 22  2021 secrets.yaml
drwxr-xr-x  2 umbrel umbrel      4096 Jun 11 21:06 .storage
drwxr-xr-x  2 umbrel umbrel      4096 Aug 22  2021 tts
drwxr-xr-x  2 umbrel umbrel      4096 Aug 22  2021 www
-rw-r--r--  1 umbrel umbrel    475136 Jun 11 21:03 zigbee.db
`
automations.yaml is overwritten by root. This might be the issue why it won't load

**Comment by jardous** [comment]
@ShonP40 yes, of course. I am reporting it, because it stopped working after recent upgrade. It was working great for months.

**Comment by ShonP40** [comment]
Well, I just installed Umbrel and seems like 0.5 uses the root user by default now

**Comment by jardous** [comment]
ouch! I wouldn't expect that from SW that I trust with my money... ;)

**Comment by jardous** [comment]
@kroese  in that case I'd not be able to see entries in my Overview page, right? But I do. 

The issue is only with the automations. If I create a new automation, it adds it to the correct automations.yaml, but it does not list it in the UI.

**Comment by jardous** [comment]
double-checked the paths. Adding invalid lines does not result in an error during Configuration validation.

Weird is, that when I add new automation in UI, I see it appended in the automations.yaml, but it does not show in the UI list...

**Comment by davidecorradini** [comment]
@jardous I'm facing the same problem right now. Did you manage to solve it?

**Comment by jardous** [comment]
yes, @davidecorradini. There were unsupported entries in the configuration.yaml

**Comment by davidecorradini** [comment]
@jardous my configuration file does not have any other entry... I think it is being ignored by HA at all (if I put wrong stuff in it, HA does not complain...)

**Comment by jardous** [comment]
I stopped running HA as an Umbrel app and running it directly as a docker container...

**Comment by davidecorradini** [comment]
Yeah, it seems to be the better thing to do. Thanks!
